### PR TITLE
Adding initial setup code for report manager service

### PR DESCRIPTION
**Pull request by Dmaddu**
Signed-off-by: Durga Sarat Chandra Maddu <durgasaratchandra.maddu@progress.com>

Issue: https://github.com/chef/automate/issues/5675

### :white_check_mark: Checklist

**All PRs** must tick these:

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [x] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [x] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [x] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [x] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [x] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [x] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [ ] Tests added/updated? (all new code needs new tests)
- [ ] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

File: components/report-manager-minio-gateway/habitat/hooks/run [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!{{pkgPathFor "core/bash"}}/bin/bash
```
Comment (Dmaddu) [inline components/report-manager-minio-gateway/habitat/hooks/run:1]: This is a basic setup script and with hard-coded credentials to unblock the actual development.
Production ready script will be added as part of issue#5787

File: components/report-manager-minio-gateway/habitat/hooks/health-check [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!{{pkgPathFor "core/bash"}}/bin/bash
+
+exec 2>&1
+
+#curlOpts="-sS --fail --max-time 2"
```
Comment (kalroy) [inline components/report-manager-minio-gateway/habitat/hooks/health-check:5]: Is there a reason why we are keeping these commented lines?

**Review by kalroy — APPROVED** [review]
LGTM!!

**Comment by Dmaddu** [comment]
@kalroy @rickmarry @punitmundra @vivek-yadav  
I am merging these changes to unblock the development, though the changes are merged, please feel free to provide review comments, so that I can incorporate them in other PRs.

File: components/report-manager-minio-gateway/habitat/hooks/health-check (hunk 2 above)
Comment (Dmaddu) [reply components/report-manager-minio-gateway/habitat/hooks/health-check:5]: @kalroy  These are just to the reminders of pending work to me.

File: components/report-manager-service/cmd/report-manager-service/report-manager-service.go [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+	if err := initConfig(); err != nil {
+		return errors.Wrap(err, "failed to load config")
+	}
+
+	//Wait till report-manager-minio-gateway is up and running
+	time.Sleep(30 * time.Second)
+
+	endpoint := "127.0.0.1:10197"
+	accessKeyID := "minioadmin"
```
Comment (rickmarry) [inline components/report-manager-service/cmd/report-manager-service/report-manager-service.go:85]: should accessKeyID and secretAccessKey be externalized?

**Pull request merged**